patches/server/0919-Optimize-VarInts.patch

```diff
@@ -0,0 +1,59 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: astei <andrew@steinborn.me>
+Date: Sat, 1 Oct 2022 09:56:47 +0200
+Subject: [PATCH] Optimize VarInts
+
+
```

Comment:
Please link the optimization 
https://github.com/PaperMC/Paper/pull/6957#issuecomment-985250854

would be nice to have some context what this is doing and why